commands/drawCommand.js

```diff
@@ -2,7 +2,10 @@ import axios from "axios";
 import { MessageAttachment } from "discord.js";
 import db from "../db.js";
 
-export const RARITIES = [
+let timeDifference;
+const DRAW_WAIT_TIME = 600000; // 600000 is 10 minutes in milliseconds
+const COST_OF_DRAW = 1.5;
```

Comment:
could use a comment that this is in dollars -- some people would also say that alternatively it could be in the name `DOLLAR_COST_OF_DRAW` or `COST_OF_DRAW_IN_DOLLARS`